lib/cast-image/EmbedImg.tsx

```
@@ -9,16 +9,104 @@ export default function EmbedImg({
+        return (
+          <img
+            src={url}
+            width={w}
+            height={h}
+            style={{
+              maxWidth: "50%",
+              maxHeight: "50%",
+              borderRadius: "20px",
+              objectFit: "cover",
+            }}
+          />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)